### PR TITLE
Forecast updates

### PR DESCRIPTION
**Pull request by limnoliver**

### CODE REVIEW

File: in_text/text_02_observations.yml [hunk 1]
```diff
@@ -22,11 +17,40 @@ cross-cites:
...
+    pubdate: XXXX
+    link: XX
 
 entities:
   -
-    data-name: temperature_observations.zip
-    data-description: Temperature observation data with sites matched to 456 model river segments in the Delaware River Basin.
+    data-name: temperature_observations{subset}.zip
+    data-description: >- 
+      Daily ater temperature observations with sites matched to 456 model river segments in the Delaware River Basin.
```
Comment (hcorson-dosch-usgs) [inline in_text/text_02_observations.yml]: Tiny typo - should be `daily water temperature`

File: in_text/text_02_observations.yml [hunk 2]
```diff
@@ -22,11 +17,40 @@ cross-cites:
...
+      Files represent various subsets of similar data, and are either from all 456 segments (temperature_observations.zip), a subset of XX segments above the
+      Delaware River at Lordville, NY (temperature_observations_lordville.zip), or at the 5 priority forecast sites.
+      Water temperature observations were compiled from NWIS, EcoSHEDS, and WQP. If multiple sites were matched to a single
+      river reach and had measurements on the same day, the values were averaged.
+   
+      Temperature observation for five priority forecasting sites in the Delaware River Basin
```
Comment (hcorson-dosch-usgs) [inline in_text/text_02_observations.yml]: Should observation be plural?

File: 2_observations.yml [hunk 3]
```diff
@@ -18,29 +18,38 @@ targets:
       - out_data/reservoir_releases.csv
```
Comment (hcorson-dosch-usgs) [inline 2_observations.yml]: These reservoir releases file names need to be updated here as well

File: in_text/text_02_observations.yml [hunk 4]
```diff
@@ -191,54 +182,39 @@ entities:
...
       data-max: NA
       data-units: NA 
+
   -
-    data-name: reservoir_releases.csv
-    data-description: Daily reservoir releases from the Pepacton, Neversink, and Cannonsville reservoirs.
+    data-name: reservoir_releases_by_type{_subset}.csv
+    data-description: >-
+      Daily reservoir releases from the Pepacton, Neversink, and Cannonsville reservoirs (reservoir_releases_by_type.csv) or
+      from only those reservoirs above the Delaware River at Lordville, NY (reservoir_releases_by_type_lordville.zip).
```
Comment (hcorson-dosch-usgs) [inline in_text/text_02_observations.yml]: I think the lordville subset also is a .csv not .zip, right?

File: 2_observations.yml (hunk 3 above)
Comment (limnoliver) [reply 2_observations.yml]: good catch. Fixed.

**Review by hcorson-dosch-usgs — APPROVED** [review]
Looks good, Sam! Once you've merged this in I'll pull it down and add the NLDAS metadata

**Pull request merged**